App/controllers/upload-file.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+var bluebird = require('bluebird');
+var xlsx = require('xlsx');
+var mongoose = require('mongoose');
+
+var awsUpload = require('../services/aws-services.js');
+var dbServices = require('../services/db-services.js');
+var jsonDataModel = require('../models/json-file-schema.js');
+
+var k = require('../../config/constants.js');
+mongoose.connect(k.DB.URL);
+var db = mongoose.connection;
+db.on('error', console.error.bind(console, 'Connection error : '));
+
+var processFileUpload = function (uploadFile) {
+  var workbook = xlsx.readFile(uploadFile.path);
+  bluebird.map(workbook.SheetNames, function (item) {
+    var jsonArray = xlsx.utils.sheet_to_json(workbook.Sheets[item]);
+    return bluebird.map(jsonArray, function (obj) {
+
+      if (obj && obj[k.CATEGORY_ID] && obj.Category && obj[k.META_TITLE] && obj[k.META_DESCRIPTION]) {
+        var newJsonData = {
```

Comment:
As per schema, meta title and meta descriptions are not mandatory. Please update schema if these keys are mandatory.